framework/src/base/FEProblemBase.C

```diff
@@ -3270,21 +3270,35 @@ FEProblemBase::postExecute()
 }
 
 void
-FEProblemBase::advanceMultiApps(ExecFlagType type)
+FEProblemBase::incrementMultiAppTStep(ExecFlagType type)
 {
   const auto & multi_apps = _multi_apps[type].getActiveObjects();
 
   if (multi_apps.size())
   {
-    _console << COLOR_CYAN << "\nAdvancing MultiApps" << COLOR_DEFAULT << std::endl;
+    _console << COLOR_CYAN << "\nAdvancing MultiApps t_step" << COLOR_DEFAULT << std::endl;
```

Comment:
I'm not sure if I'm crazy about these new messages. We have two capitalized words followed by this cryptic "t_step" thing. Can you spell it out, or just leave it off like it was before? The other issue with "t_step" is that sometimes the multiapps take several steps and sometimes they take one.